lightning/src/util/config.rs

```diff
@@ -48,6 +48,30 @@ pub struct ChannelHandshakeConfig {
 	/// Default value: 1. If the value is less than 1, it is ignored and set to 1, as is required
 	/// by the protocol.
 	pub our_htlc_minimum_msat: u64,
+	/// Sets the percentage of the channel value we will cap the total value of outstanding inbound
+	/// HTLCs to.
+	///
+	/// This can be set to a value between 1-100, where the value corresponds to the percent of the
+	/// channel value in whole percentages.
+	///
+	/// Note that:
+	/// * If configured to another value than the default value 10, any new channels
+	/// created with the non default value will cause versions of LDK prior to 0.0.100 to refuse to
+	/// read the ChannelManager.
+	///
+	/// * This caps the total value for inbound HTLCs in-flight only, and there's currently
+	/// no way to configure the cap for the total value of outbound HTLCs in-flight.
+	///
+	/// * The online requirements for ensuring the safety of HTLC-encumbered funds are
+	/// different from the non-HTLC-encumbered funds. This makes this an important knob to restrict
+	/// exposure to loss due to being offline for too long.
+	/// See [`ChannelHandshakeConfig::our_to_self_delay`] and [`ChannelConfig::cltv_expiry_delta`]
+	/// for more information.
```

Comment:
suggestion: s/The online requirements for ensuring/The requirements for your node being online to ensure (just wanna make sure it's clear)